main/main.go

```diff
@@ -0,0 +1,260 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+	"github.com/stackrox/k8s-istio-cve-pusher/utils"
+)
+
+const (
+	logDataFeedName      = "data feed name"
+	kubernetesBucketDir  = "k8s"
+	istioBucketDir       = "istio"
+	cveFileExt           = ".json"
+	k8sCveFileName       = "NVDk8sCVEs.json"
+	k8sCveHashFileName   = "k8sCVEsHash"
+	istioCveFileName     = "NVDistioCVEs.json"
+	istioCveHashFileName = "istioCVEsHash"
+)
+
+type component int
+
+const (
+	k8s = iota
```

Comment:
`k8s component = iota` (It works either way, but this way we're being more explicit about what the type of the enum is)